modules/i18n/src/translations/en-US/portals/actions.ts

```diff
@@ -113,10 +113,10 @@ export const actions: actionsNS = {
                 create: "Once added, these secrets will not be displayed. You will only be able to reset them.",
                 update: "Once updated, these secrets will not be displayed. You will only be able to reset them again."
             },
-            label: "Authentication Type",
-            placeholder: "Select Authentication Type",
+            label: "Authentication Scheme",
```

Comment:
How about "Authentication Method" instead? That's more comprehensive than "Scheme" IMO.

Reply:
This was suggested by @johannnallathamby and @malithie as this is the term that is used in other similar places